qt/scientific_interfaces/test/ISISReflectometry/Runs/RunsPresenterTest.h

```diff
@@ -983,17 +992,26 @@ class RunsPresenterTest : public CxxTest::TestSuite {
     ON_CALL(m_view, getSearchInstrument()).WillByDefault(Return(instrument));
   }
 
+  void expectGetUpdateInterval(std::string const &updateInterval) {
+    ON_CALL(m_view, getLiveDataUpdateInterval())
```

Comment:
As we discussed offline, I think this and the existing `expectSearchInstrument` function could use EXPECT_CALL here instead to make them more consistent with what the function names suggest. Then the EXPECT_CALL can be removed from `testStartMonitorUpdatesView`